### DISCUSSION TITLE
Payload Roadmap

### DISCUSSION THREAD

**Discussion by denolfe** [payloadcms/payload#1542] [Roadmap]
Use the following links to view the current roadmap:

[Roadmap Priority 1](https://github.com/payloadcms/payload/discussions?discussions_q=label%3Aroadmap-priority-1)
[Roadmap Priority 2](https://github.com/payloadcms/payload/discussions?discussions_q=label%3Aroadmap-priority-2)
[Roadmap Priority 3](https://github.com/payloadcms/payload/discussions?discussions_q=label%3Aroadmap-priority-3)

Feel free to create a new feature request or [upvote any existing feature request](https://github.com/payloadcms/payload/discussions/categories/feature-requests-ideas) in order to get it on our radar and prioritize for a future release accordingly.

**Comment by ds-onv** [comment]
Everyone reading this - links are currently broken/have been changed:

[Priority 1](https://github.com/payloadcms/payload/discussions?discussions_q=label%3A%22roadmap%3A+p1%22+is%3Aopen)
[Priority 2](https://github.com/payloadcms/payload/discussions?discussions_q=label%3A%22roadmap%3A+p2%22+is%3Aopen)
[Priority 3](https://github.com/payloadcms/payload/discussions?discussions_q=label%3A%22roadmap%3A+p3%22+is%3Aopen)
